modules/core/src/main/java/org/apache/ignite/internal/maintenance/MaintenanceProcessor.java

```diff
@@ -231,6 +243,14 @@ else if (isMaintenanceMode()) {
                     "Restart the node to get it back to normal operations.");
             }
         }
+
+        if (isMaintenanceMode() && autoShutdown) {
+            if (log.isInfoEnabled())
+                log.info("Starting thread which will check active maintenance tasks. " +
```

Comment:
I would rephrase it into something like: 
`Starting a thread that will shut down the node when all maintenance tasks are completed.`